### PR TITLE
[Operation System] Race Condition

### PR DESCRIPTION
**Pull request by kimgwon**
## Check List

- [x] I created file and check spell again
- [x] I linked File to README.md

### CODE REVIEW

File: CS Deep Dive/Operating System/Race Condition.md [hunk 1]
```diff
@@ -1,40 +1,45 @@
...
-> `Busy Waiting` : Critical Section 진입을 기다리면서 계속 CPU와 메모리를 사용
+\*[Semaphore & Mutex 자세히 보기](https://github.com/da-in/tech-interview-study/blob/main/CS%20Deep%20Dive/Operating%20System/세마포어(Semaphore)%20&%20뮤텍스(Mutex).md)  
 
 ----
 ## Reference
 - https://iredays.tistory.com/125
 - https://velog.io/@klloo/%EC%9A%B4%EC%98%81%EC%B2%B4%EC%A0%9C-%EA%B2%BD%EC%9F%81-%EC%83%81%ED%83%9C-Race-Condition
 -https://systorage.tistory.com/entry/CS-%EC%9A%B4%EC%98%81%EC%B2%B4%EC%A0%9C-%ED%94%84%EB%A1%9C%EC%84%B8%EC%8A%A4-%EB%8F%99%EA%B8%B0%ED%99%94
 - https://rebro.kr/176
+- 명품 운영체제
```
Comment (da-in) [inline CS Deep Dive/Operating System/Race Condition.md:45]:
```suggestion
- 명품 운영체제, 황기태 저, 생능출판사, 2023 
```

File: CS Deep Dive/Operating System/Race Condition.md [hunk 2]
```diff
@@ -1,40 +1,45 @@
...
+<br>
 
-> `Busy Waiting` : Critical Section 진입을 기다리면서 계속 CPU와 메모리를 사용
+\*[Semaphore & Mutex 자세히 보기](https://github.com/da-in/tech-interview-study/blob/main/CS%20Deep%20Dive/Operating%20System/세마포어(Semaphore)%20&%20뮤텍스(Mutex).md)  
 
 ----
 ## Reference
 - https://iredays.tistory.com/125
 - https://velog.io/@klloo/%EC%9A%B4%EC%98%81%EC%B2%B4%EC%A0%9C-%EA%B2%BD%EC%9F%81-%EC%83%81%ED%83%9C-Race-Condition
 -https://systorage.tistory.com/entry/CS-%EC%9A%B4%EC%98%81%EC%B2%B4%EC%A0%9C-%ED%94%84%EB%A1%9C%EC%84%B8%EC%8A%A4-%EB%8F%99%EA%B8%B0%ED%99%94
```
Comment (da-in) [inline CS Deep Dive/Operating System/Race Condition.md:43]:
```suggestion
- https://velog.io/@klloo/운영체제-경쟁-상태-Race-Condition
- https://systorage.tistory.com/entry/CS-운영체제-프로세스-동기화
```

**Review by da-in — APPROVED** [review]
깔끔해요 👍🏻

**Review by lalabulla — APPROVED** [review]
좋아요🙌🏻

**Pull request merged**